packages/react-native/cli.js

```diff
@@ -129,7 +134,7 @@ async function main() {
   if (isNpxRuntime && !process.env.SKIP && currentVersion !== HEAD) {
     try {
       const latest = await getLatestVersion();
-      if (latest !== currentVersion) {
+      if (parseVersion(latest) > parseVersion(currentVersion)) {
```

Comment:
```suggestion
      if (parseVersion(latest) !== parseVersion(currentVersion)) {
```

Reply:
That would defeat the purpose of this change. Nightlies shouldn't trigger this warning.